### ISSUE TITLE
Media Browser doesn't display popup to select a media source for user profile photos in MODX 3.0

### ISSUE THREAD

**Issue by SnowCreative** [modxcms/revolution#16510]
### Summary
Media browser that popups up when selecting a photo for a user profile only displays the default media source

### Step to reproduce
In MODX 3 site, edit your profile, and click the icon to select a user photo.

### Observed behavior
There is no popup to select a media source. Only the folder for the default media source is displayed.
![Screen Shot 2024-01-12 at 1 20 24 PM](https://github.com/modxcms/revolution/assets/2528374/8a3a269b-23a3-42e4-8179-8e9df269120f)

### Expected behavior
The Media Browser popup to select a media source should be displayed, just like it is everywhere else the media browser is used.
![Screen Shot 2024-01-12 at 1 22 10 PM](https://github.com/modxcms/revolution/assets/2528374/8310be3d-f422-479f-84e6-5b90a1992d5c)

### Environment
MODX 3.0.4-pl

**Comment by smg6511** [comment]
Ok, so after a little research on this I found that the behavior has always been this way dating back to the introduction of profile photos nearly [10 years ago](https://github.com/modxcms/revolution/commit/7220c7ab7e4d5d6bbaaa863b6851e767edfd8489). The source combo was purposely hidden and a system setting (`photo_profile_source`) was added into the conditional that defines which source the media browser should use (in the context of profile photo insertion):
```javascript
{
    id: 'modx-user-photo'
    ,fieldLabel: _('user_photo')
    ,name: 'photo'
    ,xtype: 'modx-combo-browser'
    ,hideFiles: true
    ,source: MODx.config['photo_profile_source'] || MODx.config.default_media_source
    ,hideSourceCombo: true
    ,anchor: '100%'
}
```

@rtripault was the author of the original code and may be able to shed some light on that decision (_maybe_ ... it's been an awfully long time!). 

Before proceeding on any changes, the question arises: Is this a bug, a missing feature, or a move made (to limit the source to the two settings-defined options) for good reason that we should stick with?

**Comment by SnowCreative** [comment]
Ah, I was hoping there was a system setting but couldn't find it. That kind of makes sense, but I don't know why just using the normal media manager setup wouldn't work. I assume access to media sources would be limited by permissions set on each source, as it is in every other context of using the media manager. If anything is preventing that from being the case for profile photos, then I can see why a system setting would be needed. What would be nicest is to make the system setting optional: if something is entered there, then only show that media source; and if it's empty then show the full list in the browser.

**Comment by rtripault** [comment]
That's an old one indeed! 

Not sure what was the old me thinking back then... i guess ease of being able to render pictures in the frontend then... but having ability to pick the source definitely makes sense!

With current implementation, you might end up having a media source configured which some user might not have access too.

**Comment by smg6511** [comment]
Alright, sounds like there's no reason _not_ to add the sources dropdown. It'll involve a handful of adjustments and additions, which I could dig into in the next week or two...

**Comment by smg6511** [comment]
After thinking on this a little more and batting it around with @opengeek, I've concluded that the best course of action would be: 

1. Show a disabled source dropdown in the media browser. This would keep the UI more consistent; I could also add a little hint via a tooltip or simply via a title attribute re the ability to customize the source via the `photo_profile_source` system setting. (That setting has always been _available_ for choosing a photo source other than the `default_media_source`, but never installed.); and
2. Add (install) the `photo_profile_source` setting into the predefined system settings. Right now, that special source is taken into account but, because it was never made available otherwise, one would never know it was there to take advantage of. 

This should offer plenty of granularity via user group, context, and/or user settings.

BTW, the reason to _not_ enable the sources dropdown: If we did that, we'd need to persist that source via a new column in the user profile; not a big deal, but it's really not the type of data we want to be storing there.
